ogcapi-features/openapi.yaml

```diff
@@ -31,7 +31,7 @@ paths:
             (link relations `service-desc` and `service-doc`),
             the Conformance declaration (path `/conformance`,
             link relation `conformance`), and the Feature
-            Collections (path `/collections`, link relation
```

Comment:
Maybe also check whether the original in ogcapi-features has this issue and fix it there, too... Makes alignment in the future easier for us...

Reply:
reverted this and the other one in ogcapi